function.go

```diff
@@ -68,6 +68,43 @@ func NewFunction(root string) (f Function, err error) {
 	return
 }
 
+// LoadFunction loads an existing Function project from disk at 'path'
+// If no Function project exists at the path location, an error is returned.
+func LoadFunction(root string) (f Function, err error) {
+	f, err = NewFunction(root)
+	if err != nil {
+		return
+	}
+	// Ensure that the configuration is actually pointing to a Function project directory.
+	if !f.Initialized() {
+		err = fmt.Errorf("No Function project found at %v", root)
+	}
+	return
+}
+
+// OverrideImage overwrites (or sets) the value of the Function's .Image
+// property, which preempts the default functionality of deriving the value as:
+// Deafult:  [config.Repository]/[config.Name]:latest
+func (f Function) OverrideImage(image string) error {
+	if image == "" {
+		return nil
+	}
+	f.Image = image
+	return f.WriteConfig()
+}
```

Comment:
FYI, in casd you are wondering why this function was in the CLI codebase not the "core" logic: it was because this overriding is specifically an override of that image prior to the invocation of the client.  In away mutating the state of the function prior to actually invoking the client.  This is a way to provide the ability to override the image name via a flag without "muddying up" the client library such that it can instead always trust what is in .faas.yaml.